packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Okta Verify (Push/OTP) integration guide
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+<StackSelector />
+
+This guide shows you how to integrate Okta Verify into your app using the embedded SDK.
+
+---
+**Learning outcomes**
+
+* Understand the Okta Verify enrollment and challenge flows
+* Learn step-by-step how to integrate Okta Verify into your authentication use case
+
+**What you need**
+
+* <StackSnippet snippet="whatyouneedsdk" />
+* <StackSnippet snippet="whatyouneedorg" />
+
+**Sample code**
+
+* <StackSnippet snippet="samplecode" />
+
+**Related use cases**
+
+<StackSnippet snippet="relatedusecases" />
+
+---
+
+## Overview
+
+...
+
+<!--
+<div class="common-image-format">
+
+![Diagram showing the WebAuthn enrollment flow](/img/authenticators/authenticators-webauthn-flow-overview.png)
+
+</div>
+
+As the service provider, you can provide WebAuthn support to your users by enabling it in your Okta org and building out support for it in your application using the Embedding SDK.
```

Comment:
Is the subject of this sentence WebAuthn support or the service provider? What is "service provider" in this context... a set of APIs on a server that's run by the organization? What is the "application" and how does that related to "service provider"? How does WebAuthn related to Okta Verify... does Okta Verify implement it?

I think that it may be something like:
Provide WebAuthn support in your application first by enabling Okta Verify in your Okta org, and then by using the Embedded SDK to add the functionality to your app.